src/main/resources/liquibase/036-updateDB.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet author="pegah" id="covid-testResults">
+        <addColumn tableName="ddp_kit">
+            <column name="test_result" type="VARCHAR(100)" afterColumn="needs_approval">
+            </column>
+        </addColumn>
+        <addColumn tableName="ddp_kit">
+            <column name="result_date" type="VARCHAR(100)" afterColumn="test_result">
```

Comment:
I only see code to write into test_result. is result_date needed?

Reply:
nope gonna get removed ( I think I forgot to push that )